xapian-letor/tests/testdata/queryone.txt

```diff
@@ -1 +1 @@
-110001 'tigers'
+110001 'tigers score'
```

Comment:
This isn't really related to the rest of the changes, so I don't think should be here. More worryingly, a change like this should really have broken one or more of the other tests. I suspect it hasn't because `score` isn't a term in the databases used or something, but can you double check that there's nothing wrong with the other tests? (They could be under-testing the desired output. for instance.)

Reply:
fixed in a fixup commit.